sklearn/preprocessing/imputation.py

```diff
@@ -102,11 +103,21 @@ class Imputer(BaseEstimator, TransformerMixin):
         - If `axis=0` and X is encoded as a CSR matrix;
         - If `axis=1` and X is encoded as a CSC matrix.
 
+    add_missing_indicator : boolean, optional (default=False)
+        If True, the transformed X will have an appended indicator matrix
```

Comment:
I'm not happy with the use of "appended matrix" so much as "appended features". How about the following wording?

```
If True, the transformed ``X`` will have binary indicator features appended. These correspond to input features with missing values, marking which elements have been imputed.
```